config/initializers/mobility.rb

```diff
@@ -0,0 +1,135 @@
+Mobility.configure do |config|
+  # Sets the default backend to use in models. This can be overridden in models
+  # by passing +backend: ...+ to +translates+.
+  config.default_backend = :key_value
```

Comment:
This is for the polymorphic strategy -- the one I'd planned before adding this gem, and the one the gem docs recommend.